packages/select/src/SingleSelect/SingleSelectTrigger.tsx

```diff
@@ -0,0 +1,58 @@
+import type { SingleSelectState } from './useSingleSelectState';
+import type { SharedSelectTriggerProps } from '../types';
+import type { Node } from '@react-types/shared';
+import type { ReactNode } from 'react';
+
+import { ExpandMore } from '@launchpad-ui/icons';
+import cx from 'classix';
+
+import styles from '../styles/Select.module.css';
+
+type SingleSelectTriggerChildrenState<T extends object> = SingleSelectState<T> & {
+  selectedItem: Node<T>;
+};
+
+type SingleSelectTriggerProps<T extends object> = SharedSelectTriggerProps & {
+  state: SingleSelectState<T>;
+  children?: (state: SingleSelectTriggerChildrenState<T>) => ReactNode;
+};
+
+const SingleSelectTrigger = <T extends object>(props: SingleSelectTriggerProps<T>) => {
+  const { state, triggerProps, valueProps, triggerRef, children } = props;
+
+  const getRenderedSelected = () => {
+    const item = state.selectedItem;
+
+    if (!item) return 'Select option';
```

Comment:
i wonder if this placeholder text should be expose as a prop?

Reply:
Oh yes, definitely should be.